src/main/java/seedu/duke/logic/commands/gates/CalculateBooleanCommand.java

```diff
@@ -2,16 +2,18 @@
 
 import seedu.duke.DukeException;
 import seedu.duke.model.template.BooleanTemplate;
+import seedu.duke.ui.Ui;
 
 public class CalculateBooleanCommand extends BooleanCommand {
     public static final String COMMAND_WORD = "calc";
+    public static final Ui ui = new Ui();
 
     public CalculateBooleanCommand(BooleanTemplate template) {
         super(template);
     }
 
     @Override
     public void execute() throws DukeException {
-
+        ui.printMessage("The output of the current configuration is " + template.calculateOutput(0) + ".");
     }
```

Comment:
Following `CircuitCommand`, what could be done instead is to put this in an overridden `toString()` method:

```suggestion
    }
    
    @Override
    public String toString() {
        return "The output of the current configuration is " + template.calculateOutput(0) + ".\n" + super.toString();
    }
```

Reply:
Good idea! Will do that.